packages/mds-stream/kafka/stream-producer.ts

```diff
@@ -65,6 +65,7 @@ export const KafkaStreamProducer = <TMessage>(
           topic,
           messages
         })
+        return
```

Comment:
I worry that the `return` will be easy to accidentally remove. Maybe move the test/throw to the top of the method?
```ts
if (!isDefined(producer)) {
  throw new ClientDisconnectedError(ExceptionMessages.INITIALIZE_CLIENT_MESSAGE)
}
...
```